certified-connectors/Oodrive Sign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2891 @@
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "$ref": "#/definitions/ArrayOfContract"
+            }
+          }
+        },
+        "summary": "Get a list of all contracts",
+        "description": "Retrieve all contracts from the associated license.",
+        "operationId": "ListAllContracts",
+        "parameters": [
+          {
+            "name": "size",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "x-ms-summary": "Size of the returned array",
+            "description": "The maximum size of the returned array of contract.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "offset",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "x-ms-summary": "Skip first n contracts",
+            "description": "The sequence number of contract that will be skipped.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "get_properties",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "x-ms-summary": "Show contract properties",
+            "description": "If true, contract properties will be listed in the response.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "get_perimeters",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "x-ms-summary": "Show contract perimeters",
+            "description": "If true, contract perimeters will be listed in the response.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "before",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "x-ms-summary": "Created before",
+            "description": "Will list contract created before the given date.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "after",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Will list contract created after the given date.",
+            "x-ms-summary": "Created after",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "perimeter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Only with perimeter",
+            "description": "List contracts only in a given perimeter.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "status",
+            "in": "query",
+            "x-ms-summary": "Only with status",
+            "type": "array",
+            "collectionFormat": "csv",
+            "description": "Retrieve contract only with specified status",
```

Comment:
Hi @QwerioTEAM, Thank you so much for your submission. 
All parameters descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
Please fix this for all the operations